### ISSUE TITLE
Unable to skip column using JSONPropertyOrder

### ISSUE THREAD

**Issue by shankyty** [FasterXML/jackson-dataformat-csv#136]
Unable to skip column  in csv using JSONPropertyOrder

**Comment by cowtowncoder** [comment]
I don't understand: skip? How would `@JSONPropertyOrder` help? CSV is positional so you must specify all columns the precede ones you need; there's no way around that.

You can, if that helps, just bind rows as `String[]` (or `List<String>`), and then extract positions you care about.

**Comment by shankyty** [comment]
But I can specific column number using @JsonProperty("6") something like
this for csv then I do not have create schema separately before calling
mapper.read

**Comment by cowtowncoder** [comment]
@shankyty No. Properties are specified by name with that notation, and having JSON property name like "6" is completely legal (even if unlikely).

However, there is `index`:

`@JsonProperty(index=6)`

which could conceivably allow doing this. So... perhaps something like this could be done, theoretically speaking, if anyone wanted to work on this.

I could modify the title, or create a new issue, for this idea.

**Comment by shankyty** [comment]
Reply I can work on this since I need it and I think it would cleaner
approach for me too.
